store/package-store/src/storeController/prune.ts

```diff
@@ -13,7 +13,7 @@ export interface PruneOptions {
   storeDir: string
 }
 
-export async function prune ({ cacheDir, storeDir }: PruneOptions) {
+export async function prune ({ cacheDir, storeDir }: PruneOptions, force?: boolean) {
```

Comment:
Name the value here more specifically
```suggestion
export async function prune ({ cacheDir, storeDir }: PruneOptions, removeAlienFiles?: boolean) {
```